tests/test_query_visitors.py

```diff
@@ -122,47 +117,10 @@
             "OFFSET 1",
             "GRANULARITY 3600",
         ),
-        None,
         id="multiple ORDER BY",
     ),
     pytest.param(
-        Query("discover", Entity("events", None, 0.2))
-        .set_select([Column("event_id"), Column("title")])
-        .set_where([Condition(Column("timestamp"), Op.GT, NOW)])
-        .set_limit(10)
-        .set_offset(1)
-        .set_granularity(3600)
-        .set_totals(False)
-        .set_consistent(True)
-        .set_turbo(True)
-        .set_dry_run(True)
-        .set_legacy(True)
-        .set_debug(True)
-        .set_parent_api("testing")
-        .set_team("search")
-        .set_feature("sdk-testing"),
-        (
-            "MATCH (events SAMPLE 0.200000)",
-            "SELECT event_id, title",
-            "WHERE timestamp > toDateTime('2021-01-02T03:04:05.000006')",
-            "LIMIT 10",
-            "OFFSET 1",
-            "GRANULARITY 3600",
-        ),
-        [
-            ("consistent", True),
-            ("turbo", True),
-            ("debug", True),
-            ("dry_run", True),
-            ("legacy", True),
-            ("parent_api", "testing"),
-            ("team", "search"),
-            ("feature", "sdk-testing"),
-        ],
-        id="test flags",
-    ),
```

Comment:
Is there a reason this was removed?

Reply:
This is testing adding flags to a Query which is no longer allowed. This logic was added to the `test_request` file.